modules/network-observability-ebpf-manager.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-ebpf-manager_{context}"]
+= Working with eBPF manager
+The eBPF manager is a separate operator that manages all eBPF programs, thereby reducing the attack surface, and ensuring compliance, security, and preventing conflicts. Network observability leverages eBPF Manager to handle the loading of hooks, removing the need to provide the eBPF Agent with privileged mode or additional Linux capabilities such as `CAP_BPF` and `CAP_PERFMON`. The eBPF manager with network observability is only supported on 64-bit AMD architecture.
+
+//Delete comments before merging PR
+//05/02/2025: Source https://netobserv.io/posts/what-s-new-in-network-observability-1-8/ -> eBPF Manager Support.
+//Content is copy/pasted to open PR. Needs further refinement to meet Red Hat style guides.
+//According to the official Red Hat Product name spreadsheet, it is "network observability" and not "Network Observability". There does not appear to be attribute for anything related to network observability, which is something to address separately.
+//For docs.redhat, Technology Preview banners are handled separately by another team. Inserted admonition from https://redhat-documentation.github.io/supplementary-style-guide/#technology-preview-guidance. CHECK on feature name. Is it "Network observability with eBPF manager" or "eBPF manager" or something else?
+
+:FeatureName: Network observability with eBPF Manager Operator
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Operator Hub*.
+. Install *eBPF Manager*.
+. Check *Workloads* -> *Pods* in the `bpfman` namespace to make sure they are all up and running.
+. Configure the `FlowCollector` custom resource, for example:
++
+[id="network-observability-flowcollector-configuring-ebpfmanager_{context}"]
+.Example `FlowCollector` configuration
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  agent:
+    ebpf:
+      privileged: true  # required for now
+      interfaces: <1>
+        - br-ex
+      features:
+        - eBPFManager
+----
+<1> You must specify an interface so the eBPF Manager know where to attach the TCx hook.
```

Comment:
```suggestion
```
eBPF manager has auto-interface discovery capability now so no need to pass interface